### DISCUSSION TITLE
I want a speech viewer for screen readers

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#209] [Wants]
My wish would be for each screen reader to have a speech viewer tool that can be used by sighted designers and developers to debug their work. They would be able to use it like WAVE or Axe to preview how a screen reader would interpret what they’ve built. Like WAVE, it could have icons to show what each component does and syntax coloring to show errors.

Screen readers should not be a mystery to your average website builder.

<hr>

https://webwewant.fyi/wants/104/

**Comment by amundo** [comment]
This is a fantastic idea and would go a long way toward advancing the adoption of more accessible media.
